src/lxc/utils.h

```diff
@@ -296,4 +296,9 @@ int open_devnull(void);
 int set_stdfds(int fd);
 int null_stdfds(void);
 int lxc_count_file_lines(const char *fn);
+
```

Comment:
Both here and above the real function, you describe how we use it (checking for SIGRTMIN+3), but the fn itself is generic.  Better to leave these comments just saying "check for signal", as it may grow other users.